modules/services/gpg-agent.nix

```diff
@@ -41,7 +41,7 @@ in
 
       home.sessionVariables =
         optionalAttrs cfg.enableSshSupport {
-          SSH_AUTH_SOCK = "\${XDG_RUNTIME_DIR}/gnupg/S.gpg-agent.ssh";
+          SSH_AUTH_SOCK = "\$XDG_RUNTIME_DIR/gnupg/S.gpg-agent.ssh";
```

Comment:
Hmm, yeah. I suspect it will require looking for variables of the form `$XYZ` and converting to `${XYZ}` in the PAM module.